SUILesson3/SUILesson3/ContentView.swift

```diff
@@ -0,0 +1,207 @@
+                Text(Constants.notificationCount)
+                    .foregroundColor(.white)
+            }
+            Image(systemName: Constants.chevronRightImage)
+                .foregroundColor(.gray)
+        }
+    }
+
+    private func setupBluetoothView() -> some View {
+        HStack {
+            Image(systemName: Constants.blueToothImageName)
+                .foregroundColor(.black)
+                .frame(width: 30, height: 30)
+                .background(Color.white)
+                .cornerRadius(CGFloat(4))
+            Picker(Constants.bluetoothTitleText, selection: $bluetoothMode) {
+                ForEach(0..<settingsStatus.count) {
+                    Text(settingsStatus[$0])
+                }
+            }
+            .pickerStyle(.navigationLink)
+        }
+    }
+
+    private func setupMobileView() -> some View {
+        HStack {
+            Image(systemName: Constants.setupMobileImageName)
+                .foregroundColor(.white)
+                .frame(width: 30, height: 30)
+                .background(Color.blue)
+                .cornerRadius(CGFloat(4))
+            Picker(Constants.mobileTitleText, selection: $currentOperator) {
+                ForEach(0..<operatorStatus.count) {
+                    Text(operatorStatus[$0])
+                }
+            }
+            .pickerStyle(.navigationLink)
+        }
+    }
+
+    private func setupModemView() -> some View {
+        HStack {
+            Image(systemName: Constants.setupModemImageName)
+                .foregroundColor(.white)
+                .frame(width: 30, height: 30)
+                .background(Color.green)
+                .cornerRadius(CGFloat(4))
+
+            Picker(Constants.modemTitleText, selection: $modemMode) {
+                ForEach(0..<settingsStatus.count) {
+                    Text(settingsStatus[$0])
+                }
+            }
+            .pickerStyle(.navigationLink)
+        }
+    }
+
+    private var airModeView: some View {
+        HStack {
+            Image(systemName: Constants.airplaneModel)
+                .foregroundColor(.white)
+                .frame(width: 30, height: 30)
+                .background(Color.orange)
+                .cornerRadius(CGFloat(4))
+
+            Toggle(isOn: $isAviaModeEnable) {
+                Text(Constants.airmodeTitleName)
+            }
+        }
+    }
+
+    private var setupWIFIView: some View {
+        HStack {
+            Image(systemName: Constants.wifiImageName)
+                .foregroundColor(.white)
+                .frame(width: 30, height: 30)
+                .background(Color.blue)
+                .cornerRadius(CGFloat(4))
+
+            Picker(Constants.wifiTitleName, selection: $modemMode) {
+                ForEach(0..<settingsStatus.count) {
+                    Text(wifiStatus[$0])
+                }
+            }
+            .pickerStyle(.navigationLink)
+        }
+    }
+
+    private func setupVpnView() -> some View {
```

Comment:
```suggestion
    private var  vpnView(): some View {
```
по всему проекту поправь